cosmos/aggevm.json

```diff
@@ -0,0 +1,55 @@
+{
+  "rpc": "https://aggevm-rpc.web3-idea.xyz",
+  "rest": "https://aggevm-api.web3-idea.xyz",
+  "nodeProvider": {
+    "name": "Web3 Idea",
+    "email": "zhangzhishun1024@gmail.com",
+    "website":"https://web3-idea.xyz"
+  },
+  "chainId": "aggevm",
+  "chainName": "Agg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "AGG",
+    "coinMinimalDenom": "uagg",
+    "coinDecimals": 6,
+    "coinGeckoId": "agg",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/agg",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/agg",
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "agg",
+    "bech32PrefixAccPub": "aggpub",
+    "bech32PrefixValAddr": "aggvaloper",
+    "bech32PrefixValPub": "aggvaloperpub",
+    "bech32PrefixConsAddr": "aggvalcons",
+    "bech32PrefixConsPub": "aggvalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "AGG",
+      "coinMinimalDenom": "uagg",
+      "coinDecimals": 6,
+      "coinGeckoId": "agg",
```

Comment:
@zhangzhishun Could you remove coinGeckoId field? because It is invalid

Reply:
I do it. Thanks